lib/uplink/main.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplink
+
+import (
+	"context"
+
+	"storj.io/storj/pkg/identity"
+	ul "storj.io/storj/uplink"
```

Comment:
nit - I think I've seen preference to not do this shortened import name thing in the repo, unless you know otherwise.

Reply:
I needed a way to acces /uplink but since this package is called `uplink`I had to do this weirdness.

Reply:
ahh ok!

Reply:
There is no conflict in importing `uplink`.